containers/datalab/run.sh

```diff
@@ -65,6 +82,7 @@ fi
 docker run -it --entrypoint=$ENTRYPOINT \
   -p 127.0.0.1:8081:8080 \
   -v "$CONTENT/datalab:/content/datalab" \
+  $PYDATALAB_MOUNT_OPT \
```

Comment:
The old way is useful in cases where you want to build an image with custom pydatalab. For example, I want to add a new experimental feature in pydatalab (not in master yet) and I want to build a container with that feature and give it to a few users to try. Is the scenario still valid?